de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/nio/writer/debug/DebugStreamDumpWriter.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2014
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.tudarmstadt.ukp.dkpro.argumentation.nio.writer.debug;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+
+import org.apache.commons.compress.utils.IOUtils;
+import org.apache.commons.io.output.CloseShieldOutputStream;
+import org.apache.uima.UimaContext;
+import org.apache.uima.analysis_engine.AnalysisEngineProcessException;
+import org.apache.uima.fit.component.JCasAnnotator_ImplBase;
+import org.apache.uima.fit.descriptor.ConfigurationParameter;
+import org.apache.uima.jcas.JCas;
+import org.apache.uima.resource.ResourceInitializationException;
+
+/**
+ * Dumps all argument components in the {@code JCas} (similar to
+ * {@link org.apache.uima.fit.component.CasDumpWriter}). If not specified, the
+ * output goes to {@code System.out}.
+ *
+ * @author Ivan Habernal
+ */
+public final class DebugStreamDumpWriter extends JCasAnnotator_ImplBase {
```

Comment:
DKPro Core writer components based on 1.8.0-SNAPSHOT should all support writing to stdout since December. Cf.  https://github.com/dkpro/dkpro-core/issues/700